src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -754,6 +777,11 @@ private static class CliOptions {
                         + "be specified.")
         private boolean printTreeWithJavadoc;
 
+        /** Option that controls whether to print the custom AST of the file. */
+        @Option(names = {"-pc", "--printWithCustomTree"},
```

Comment:
A multi-character short name may be confusing, see https://picocli.info/#_short_posix_options

Although the library supports multi-character short names, it collects them into groups:
```bash
  -o=<outputPath>           Sets the output file. Defaults to stdout.
  -p=<propertiesFile>       Sets the property files to load.
      -pc, --printWithCustomTree=<customTreePrinterClass>
                            Prints Tree based on the provided custom class.
```

This can be changed to `-tc`:

```bash
  -t, --tree                Prints Abstract Syntax Tree(AST) of the checked file. The option cannot
                              be used other options and requires exactly one file to run on to be
                              specified.
  -T, --treeWithComments    Prints Abstract Syntax Tree(AST) with comment nodes of the checked
                              file. The option cannot be used with other options and requires
                              exactly one file to run on to be specified.
      -tc, --printWithCustomTree=<customTreePrinterClass>
                            Prints Tree based on the provided custom class.
```